types/trigger.d.ts

```diff
@@ -66,6 +67,11 @@ export function eventGrid(options: EventGridTriggerOptions): EventGridTrigger;
  */
 export function cosmosDB(options: CosmosDBTriggerOptions): CosmosDBTrigger;
 
+/**
+ * [Link to docs and examples](https://learn.microsoft.com/en-us/azure/azure-functions/functions-bindings-warmup?tabs=isolated-process&pivots=programming-language-javascript)
```

Comment:
nit: Please remove `en-us/` whenever you add a link so that it defaults to the user's browser language instead of always english

Reply:
Thanks for catching that. I fixed the url.